src/open_inwoner/openzaak/notifications.py

```diff
@@ -591,10 +591,9 @@ def send_case_update_email(
         "case_link": case_detail_url,
     }
     if status:
-        context["status_description"] = translate_single_status(
-            status.statustype.omschrijving
-        )
-
+        status_type = status.statustype
+        status_text = status_type.statustekst or status_type.omschrijving
+        context["status_description"] = translate_single_status(status_text)
```

Comment:
I note in the code above the `or` condition didn't need a temporary variable but was fine directly as argument.

Also we probably should add the default _("No data available") everywhere it makes sense (this is a legacy thing we can fix).